### PR TITLE
Update to support UMD / browser

### PR DESCRIPTION
**Pull request by brekk**
Hello,

I love the simplicity of this library; however when I try to use it via unpkg.com (https://unpkg.com/throttle-debounce@2.2.1/index.cjs.js) it ends up pointing at a commonjs implementation, which I cannot use in the browser.

In this PR I have updated the rollup configuration to support this.

While committing I ran into some issues w/r/t linting, but in files that I didn't touch — so I committed this as is, but please let me know if there's something additional I need to change.

Thanks,

Brekk

### CODE REVIEW

**Comment by brekk** [comment]
@niksy At your earliest convenience.

File: .eslintrc [hunk 1]
```diff
@@ -14,7 +14,8 @@
 			},
 			"rules": {
 				"no-console": 0
-			}
+			},
+			"excludedFiles": "*.umd.js"
```
Comment (niksy) [inline .eslintrc]: @brekk this is not needed

File: .gitignore [hunk 2]
```diff
@@ -8,3 +8,4 @@ index.cjs.js
 index.esm.js
 index.cjs.js.map
 index.esm.js.map
+*.swp
```
Comment (niksy) [inline .gitignore]: @brekk please remove this and add `index.umd.js` and `index.umd.js.map` to ignored files

**Review by niksy — CHANGES_REQUESTED** [review]

File: index.umd.js [hunk 3]
```diff
@@ -0,0 +1,151 @@
+(function (global, factory) {
```
Comment (niksy) [inline index.umd.js]: @brekk this file should be removed from Git since it’s now in `.gitignore`.

File: index.umd.js.map [hunk 4]
```diff
@@ -0,0 +1 @@
...AOJ,UAAP,KAAsB,SAA1B,EAAqC;CACpCE,IAAAA,YAAY,GAAGD,QAAf;CACAA,IAAAA,QAAQ,GAAGD,UAAX;CACAA,IAAAA,UAAU,GAAGS,SAAb;CACA;CAED;;;;;;;CAKA,WAASC,OAAT,GAAgC;CAAA,sCAAZC,UAAY;CAAZA,MAAAA,UAAY;CAAA;;CAC/B,QAAIC,IAAI,GAAG,IAAX;CACA,QAAIC,OAAO,GAAGC,IAAI,CAACC,GAAL,KAAaV,QAA3B;;CAEA,QAAID,SAAJ,EAAe;CACd;CACA,KAN8B;;;CAS/B,aAASY,IAAT,GAAgB;CACfX,MAAAA,QAAQ,GAAGS,IAAI,CAACC,GAAL,EAAX;CACAd,MAAAA,QAAQ,CAACgB,KAAT,CAAeL,IAAf,EAAqBD,UAArB;CACA;CAED;;;;;;CAIA,aAASO,KAAT,GAAiB;CAChBf,MAAAA,SAAS,GAAGM,SAAZ;CACA;;CAED,QAAIP,YAAY,IAAI,CAACC,SAArB,EAAgC;CAC/B;;;;CAIAa,MAAAA,IAAI;CACJ;;CAEDV,IAAAA,oBAAoB;;CAEpB,QAAIJ,YAAY,KAAKO,SAAjB,IAA8BI,OAAO,GAAGd,KAA5C,EAAmD;CAClD;;;;CAIAiB,MAAAA,IAAI;CACJ,KAND,MAMO,IAAIhB,UAAU,KAAK,IAAnB,EAAyB;CAC/B;;;;;;;;;;;CAWAG,MAAAA,SAAS,GAAGgB,UAAU,CACrBjB,YAAY,GAAGgB,KAAH,GAAWF,IADF,EAErBd,YAAY,KAAKO,SAAjB,GAA6BV,KAAK,GAAGc,OAArC,GAA+Cd,KAF1B,CAAtB;CAIA;CACD;;CAEDW,EAAAA,OAAO,CAACF,MAAR,GAAiBA,MAAjB,CA9FkE;;CAiGlE,SAAOE,OAAP;CACA;;CCpHD;AAEA,CAEA;;;;;;;;;;;;;;;AAcA,CAAe,mBAASX,KAAT,EAAgBqB,OAAhB,EAAyBnB,QAAzB,EAAmC;CACjD,SAAOA,QAAQ,KAAKQ,SAAb,GACJY,QAAQ,CAACtB,KAAD,EAAQqB,OAAR,EAAiB,KAAjB,CADJ,GAEJC,QAAQ,CAACtB,KAAD,EAAQE,QAAR,EAAkBmB,OAAO,KAAK,KAA9B,CAFX;CAGA;;;;;;;;;;;;;"}
```
Comment (niksy) [inline index.umd.js.map]: @brekk this file should be removed from Git since it’s now in `.gitignore`.

**Review by niksy — CHANGES_REQUESTED** [review]

**Comment by brekk** [comment]
@niksy 😅  whoops. Updated again.

**Comment by niksy** [comment]
@brekk published as 2.3.0. Thanks for the PR!

**Pull request merged**